src/main/java/mate/project/exception/RegistrationException.java

```diff
@@ -0,0 +1,18 @@
+package mate.project.exception;
+
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+
+@Getter
+public class RegistrationException extends RuntimeException {
+    private final HttpStatus status;
```

Comment:
The variable 'status' should not be part of the exception. Exceptions should not know anything about HTTP statuses.